app/com/linkedin/drelephant/exceptions/ExceptionFinder.java

```diff
@@ -64,37 +71,50 @@ public ExceptionFinder(String url, String scheduler)
 
     // create a new MRClient
     _mrClient = new MRClient();
+    logger.info("mr client");
 
     // create a new workflow client
     _workflowClient = InfoExtractor.getWorkflowClientInstance(scheduler, url);
+    logger.info("wf client");
+
 
     // get the schedulerData
     SchedulerConfigurationData schedulerData = InfoExtractor.getSchedulerData(scheduler);
+    logger.info("a");
```

Comment:
Please add meaningful message

Reply:
This wasn't meant to be added, included by mistake.